### PR TITLE
AO3-6422 Limit the number of accounts users can block

### PR DESCRIPTION
**Pull request by EchoEkhi**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [x] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6422

## Purpose

Limit the number of accounts users can block.

## Testing Instructions

See JIRA ticket.

## Credit

EchoEkhi (He/him)

### CODE REVIEW

**Comment by redsummernight** [comment]
Broken tests should be fixed on the default branch now (#4395), you should merge from it.

File: features/users/blocking.feature [hunk 1]
```diff
@@ -119,3 +119,14 @@ Feature: Blocking
       | superadmin       |
       | policy_and_abuse |
       | support          |
+
+  Scenario: Users cannot block more users than the block limit
```
Comment (brianjaustin) [inline features/users/blocking.feature:123]: I'm wondering if this should be a `spec` instead, since the check happens in the model and not frontend

**Review by brianjaustin — COMMENTED** [review]
Looks good to me! One quick remark/question on the test

**Review by sarken — COMMENTED** [review]
This has merge conflicts now. Do you happen to have time to update it? And if so, can you change the config to 2000 for now?

**Review by sarken — APPROVED** [review]

**Pull request merged**